build.rs

```diff
@@ -511,6 +526,12 @@ fn main() {
         rustcode = rustcode.replace(" asm!(", " llvm_asm!(");
     }
 
+    // Workaround and final check for https://github.com/immunant/c2rust/issues/1356
+    let mut rustcode = rustcode.replace("std::ffi::", "libc::");
+    if rustcode.contains("std::") {
```

Comment:
This can theoretically trigger on other names like `fastd` or whatever, I don't know if that will be an issue?